ecosystem/typescript/sdk_v2/src/crypto/authentication_key.ts

```diff
@@ -75,9 +76,11 @@ export class AuthenticationKey {
 
     let scheme: number;
     if (publicKey instanceof Ed25519PublicKey) {
-      scheme = AuthenticationKeyScheme.Ed25519.valueOf();
+      scheme = SigningScheme.Ed25519.valueOf();
```

Comment:
interesting, then the `valueOf()` is needed? I believe we use enum strings in different places without it.

Reply:
I believe the `valueOf()` returns the actual value or `number` of that enum. But if string is needed, we can run `toString()` to get the string version.